docs/swagger.yml

```diff
@@ -177,13 +194,49 @@ paths:
             $ref: "#/definitions/CredentialOut"
         400:
           description: "Invalid input"
+        401:
+          description: "Not authorized"
+        404:
+          description: "Credential not found"
+    patch:
+      tags:
+        - "Credential"
+      summary: "Partial update of an existing credential"
+      description: "Partial update an existing credential."
+      operationId: "partialUpdateCredential"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "cred_id"
+        in: "path"
+        description: "ID of credential to update"
+        required: true
+        type: "integer"
+        format: "int64"
+      - in: "body"
+        name: "body"
+        description: "Portion of credential object that should overlay existing credential object."
```

Comment:
I feel like the descriptions of updates should be somewhat cohesive: 
ie. full update body description says: 'Credential object to replace existing credential object'
so maybe this should say: 'Credential id to replace existing credential id' or something similar?

Reply:
Its not a replacement of credential id.